cross/wallabag/digests

```diff
@@ -0,0 +1,3 @@
+1.9.1-b.zip SHA1 48E81C712008A00B9BC8B89ABEB1A135DE2BA2DD
```

Comment:
Please set `PKG_DIST_FILE = $(PKG_NAME)-$(PKG_VERS)` in `cross/wallabag/Makefile`. Then redownload and recreate digests. This ensures the downloaded file in $distrib is named `wallabag-1.9.1-b.zip`, making it easier to identify.